react-web/src/pages/certification/Certification.tsx

```diff
@@ -247,6 +248,20 @@ const Certification = () => {
       handleErrorScenario
   )
 
+  const [render, setRendered] = useState(false);
+
+  useEffect(() => {
+    // Initial load of subscribed features
+    if (subscribedFeatures !== null) {
+      setRendered(true);
+    }
+  }, [subscribedFeatures]);
+
+  // Show loader until subscribed features is fetched
+  if (!render) {
+    return <Loader />;
+  }
+
```

Comment:
```suggestion
  // Show loader until subscribed features is fetched
  if (!subscribedFeatures) {
    return <Loader />;
  }

```

Reply:
Did this suggestion for now, as I could not reproduce the error where it showed 'You do not have any subscriptions...' while waiting for the response from /active-features API.

Reply:
Did as suggested, as with this wasn't able to reproduce the scenario where it still showed "You do not have a valid subscription to..." while it waited for response from /active-features API.